easybuild/easyconfigs/a/AOFlagger/AOFlagger-3.4.0-foss-2022a.eb

```diff
@@ -32,6 +32,9 @@ dependencies = [
     ('libxml2', '2.9.13'),
 ]
 
+# make sure the right Python is used
```

Comment:
I'm wondering if we can be more explicit, e.g. us my comment in this PR:
```
# Without this, the wrong (system) python is picked up by cmake in cases where the system python is newer than the one used in these EasyConfigs
# See PR # 19119
```
(mind the maximum line length of 120 characters though, this might exceed it in which case you'll have to spread over multiple lines)

Having this comment will help us determine in the future if this is still relevant. E.g. if we ever do implement a more permanent fix in the EasyBlock, we know that this is no longer an issue and this `configopts` can be taken out again.

Of course, same suggestion for all the other Easyconfigs in this PR :)

Reply:
Agreed, that is a more helpful comment in the future :)